rest/CMakeLists.txt

```diff
@@ -2,14 +2,24 @@
 
 etn_target(shared ${PROJECT_NAME}-rest
     SOURCES
-        conf/42_discovery.xml.in
```

Comment:
why did you remove this? This file now is unhandled by cmake at all

Reply:
in fty-automatic-group implemented in same way